src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -95,7 +95,7 @@ namespace Messaging {
 #elif CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE != 0
 #define CHIP_CONFIG_RMP_RETRANS_TABLE_SIZE (CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE)
```

Comment:
I wonder whether as a followup we should use the min of pool size and `MAX_EXCHANGE_CONTEXTS` for these.....

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/5924